bookwyrm/templates/ostatus/error.html

```diff
@@ -0,0 +1,60 @@
+                <form name="login" method="post" action="{% url 'login' %}?next={{ request.path }}?acct={{ user.remote_id }}">
+                    {% csrf_token %}
+                    <div class="columns is-variable is-1">
+                        <div class="column">
+                            <label class="is-sr-only" for="id_localname">{% trans "Username:" %}</label>
+                            <input type="text" name="localname" maxlength="150" class="input" required="" id="id_localname" placeholder="{% trans 'username' %}">
+                        </div>
+                        <div class="column">
+                            <label class="is-sr-only" for="id_password">{% trans "Password:" %}</label>
+                            <input type="password" name="password" maxlength="128" class="input" required="" id="id_password" placeholder="{% trans 'password' %}">
+                            <p class="help"><a href="{% url 'password-reset' %}">{% trans "Forgot your password?" %}</a></p>
+                        </div>
+                        <div class="column is-narrow">
+                            <button class="button is-primary" type="submit">{% trans "Log in" %}</button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+    {% elif error == 'ostatus_subscribe' %}
+    <div class="notification is-warning has-text-centered">
+        <p>{% blocktrans %}Something went wrong trying to follow <strong>{{ account }}</strong>{% endblocktrans %}</p>
+        <p>{% trans 'Check you have the correct username before trying again.' %}</p>
+    </div>
+    {% elif error == 'remote_subscribe' %}
+    <div class="notification is-warning has-text-centered">
+        <p>{% blocktrans %}Something went wrong trying to follow from <strong>{{ account }}</strong>{% endblocktrans %}</p>
+        <p>{% trans 'Check you have the correct username before trying again.' %}</p>
+    </div>
+    {% elif error == 'is_blocked' %}
+    <div class="notification is-danger has-text-centered">
+        <p>{% blocktrans %}You have blocked <strong>{{ account }}</strong>{% endblocktrans %}</p>
+    </div>
+    {% elif error == 'has_blocked' %}
+    <div class="notification is-danger has-text-centered">
+        <p>{% blocktrans %}<strong>{{ account }}</strong> has blocked you{% endblocktrans %}</p>
+    </div>
+    {% elif error == 'already_following' %}
+    <div class="notification is-success has-text-centered">
+        <p>{% blocktrans %}You are already following <strong>{{ account }}</strong>{% endblocktrans %}</p>
+    </div>
+    {% elif error == 'already_requested' %}
+    <div class="notification is-success has-text-centered">
+        <p>{% blocktrans %}You have already requested to follow <strong>{{ account }}</strong>{% endblocktrans %}</p>
+    </div>
+    {% endif %}
+</div>
+<div class="block is-pulled-right">
+    <a href="" onclick="closeWindow()">Close window</a>
```

Comment:
Accessibility-wise, I'd suggest a button instead of the anchor.

```suggestion
    <button type="button" onclick="closeWindow()">Close window</button>
```

Reply:
I have a button to commit these suggestions into the PR and I assume you do too @hughrun, but if for some permissions reason you don't, let me know

Reply:
I do! Does that commit to `hughrun:remote-follow`? I don't want to mess up the git branches if it commits directly into `bookwyrm:remote-follow` or something.

Reply:
I have no idea! I'd guess it commits it to the main repo and you'd have to pull to your local branch from upstream?

Reply:
Let's find out haha